langchain/document_loaders/html_bs.py

```diff
@@ -27,28 +29,17 @@ def __init__(
                 "bs4 package not found, please install it with " "`pip install bs4`"
             )
 
+        self.parser = BSHTMLParser(bs_kwargs=bs_kwargs)
         self.file_path = file_path
         self.open_encoding = open_encoding
-        if bs_kwargs is None:
-            bs_kwargs = {"features": "lxml"}
-        self.bs_kwargs = bs_kwargs
 
     def load(self) -> List[Document]:
-        from bs4 import BeautifulSoup
+        """Load all documents eagerly."""
+        return list(self.lazy_load())
```

Comment:
could this be implemented on BaseLoader?

Reply:
maybe not super clean, since if users don't want to implement lazy_load they need to overwrite load

Reply:
Could do either way -- one possibility is another base class that provides a default implementation